src/evops/benchmark/association.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2022, Pavel Mokeev, Dmitrii Iarosh, Anastasiia Kornilova
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Optional, Any, Callable
+from nptyping import NDArray
+
+import numpy as np
+
+
+def __quantitative_planes(
```

Comment:
Not clear name (should be smth with `ratio`, I suppose -- `correctly_matched_ratio`?), let's add comments at least what are those dicts, also not clear